website/content/docs/other-specifications/volume/host.mdx

```diff
@@ -0,0 +1,286 @@
+
+- `host_path` `(string)` - The path on disk where the volume exists. You should
+  set this only for volume registration. It will be ignored for volume creation.
+
+- `name` `(string: <required>)` - The name of the volume, which will be used as
+  the [`volume.source`][volume_source] field in job specifications that claim
+  this volume. Host volume names must be unique per node. Names are visible to
+  any user with `node:read` ACL, even across namespaces, so they should not be
+  treated as sensitive values.
+
+- `namespace` `(string: <optional>)` - The namespace of the volume. This field
+  overrides the namespace provided by the `-namespace` flag or `NOMAD_NAMESPACE`
+  environment variable. Defaults to `"default"` if unset.
+
+- `node_id` `(string)` - A specific node where you would like the volume to be
+  created (refer to the [volume placement](#volume-placement) section). Optional
+  for volume creation but required for volume registration.
+
+- `node_pool` `(string: <optional>)` - A specific node pool where you would like
+  the volume to be created (refer to the [volume placement](#volume-placement)
+  section). Optional for volume creation or volume registration. If you also
+  provide `node_id`, the node must be in the provided `node_pool`.
+
+- `parameters` <code>(map<string|string>:nil)</code> - An optional key-value map
+  of strings passed directly to the plugin to configure the volume. The details
+  of these parameters are specific to the plugin.
+
+- `plugin_id` `(string)` - The ID of the [dynamic host volume
+  plugin][dhv_plugin] that manages this volume. Required for volume
+  creation.
+
+- `type` `(string: <required>)` - The type of volume. Must be `"host"` for
+  dynamic host volumes.
+
+## Differences Between Create and Register
+
+Several fields are set automatically by Nomad or the plugin when `volume create`
+or `volume register` commands (or the equivalent APIs) are successful and you
+should not set their values if they are not supported by the operation.
+
+In **volume creation** you must set the [`plugin_id`](#plugin_id) field. The
+[`capacity`](#capacity) and [`host_path`](#host_path) fields are ignored.
+
+In **volume registration** you must set the [`node_id`](#node_id) and
+[`host_path`](#host_path) fields. The [`plugin_id`](#plugin_id) and
+[`constraint`](#constraint) fields are ignored. The [`node_pool`](#node_pool),
+[`capacity_max`](#capacity_max), and [`capacity_min`](#capacity_min) fields are
+ignored but must be consistent if set; the `node_pool` must match the node set
+by `node_id`, and the minimum capacity must be less than the `capacity_max` and
+`capacity`.
+
+## Volume Placement
+
+The `volume create` command will create the volume on a single node. If
+`node_id` is set, the volume will be created on that node. The `node_pool` must
+be unset or match that node. Otherwise, the Nomad server will iterate over the
+available nodes and place the volume on the first available node based on the
```

Comment:
```suggestion
The `volume create` command creates the volume on a single node. If
`node_id` is set, the volume is created on that node. The `node_pool` must
be unset or match that node. Otherwise, the Nomad server iterates over the
available nodes and place the volume on the first available node based on the
```